spec/ddtrace/workers/polling_spec.rb

```diff
@@ -13,7 +13,7 @@
 
     describe '#perform' do
       subject(:perform) { worker.perform }
-      after { worker.stop(true, 0) }
+      after { worker.stop(true, 5) }
```

Comment:
0 means "don't wait any time" (aka non-blocking) here. But we actually want to wait for the thread the stop before this tests is considered finished.